Firebase/InAppMessaging/Public/FIRInAppMessagingRendering.h

```diff
@@ -27,7 +27,8 @@ typedef NS_ENUM(NSInteger, FIRInAppMessagingDisplayMessageType) {
 
 typedef NS_ENUM(NSInteger, FIRInAppMessagingDisplayTriggerType) {
   FIRInAppMessagingDisplayTriggerTypeOnAppForeground,
-  FIRInAppMessagingDisplayTriggerTypeOnAnalyticsEvent
+  FIRInAppMessagingDisplayTriggerTypeOnAnalyticsEvent,
+  FIRInAppMessagingDisplayTriggerTypeProgrammatic
```

Comment:
just for sanity check, these enums are private/internal to the fiam sdk right? (They're not being passed to the display sdk since its only triggering condition vs message content/display/model info)